src/utils/decisions.ts

```diff
@@ -0,0 +1,22 @@
+import { DecisionDialogValues } from '~common/ColonyDecisions/DecisionDialog/DecisionDialog';
```

Comment:
We should add `DecisionDialogValues` to the `index` file of the folder, to make this import cleaner